providers/dns/transip/transip_test.go

```diff
@@ -127,7 +183,17 @@ func TestNewDNSProvider(t *testing.T) {
 				EnvAccountName:    "johndoe",
 				EnvPrivateKeyPath: "./fixtures/non/existent/private.key",
 			},
-			expected: "transip: could not open private key: stat ./fixtures/non/existent/private.key: no such file or directory",
+			expected: func() string {
```

Comment:
This feels like a hack. Other suggestions are more than welcome. I noticed this because I am working on a windows machine while working from home.

Reply:
instead of doing a hack, I think you can check the error type instead of the message.

Reply:
I've changed the tests to use `errors.is` for this specific check